command.go

```diff
@@ -30,6 +30,8 @@ import (
 	flag "github.com/spf13/pflag"
 )
 
+const FlagSetByCobraAnnotation = "cobra_annotation_flag_set_by_cobra"
```

Comment:
Just to clarify, this is a public const so that users could inspect the flag annotations themselves if needed?

Reply:
Honestly I wasn't too sure what was best.  Would it be useful to external programs?  I don't know.  So I ended up following the other annotations that were all public:
https://github.com/spf13/cobra/blob/7039e1fa214cfc1de404ed6540158c8fda64a758/bash_completions.go#L30-L33
But we can make it private if you prefer.

Reply:
> So I ended up following the other annotations that were all public

That sounds good! Thanks for clarifying!! happy to follow the current convention 👍🏼